jina/peapods/runtimes/jinad/__init__.py

```diff
@@ -17,26 +19,26 @@ def __init__(self, args: 'argparse.Namespace', ctrl_addr: str):
         # process.
         self.host = args.host
         self.port_expose = args.port_expose
-        self.api = PeaDaemonClient(
+        # TODO: args.timeout_ready is always set to -1 for JinadRuntime so that wait_for_success doesn't fail in Pea,
+        # so it can't be used for Client timeout. Setting `timeout` to `None` would wait forever in `DaemonClient`,
+        # which is not ideal. Setting it to 15 secs for testing.
```

Comment:
Should this TODO be fixed before merging or can be left over for later? If it stays, please add a name to it.

Reply:
changed it to NOTE, as it doesn't need any action now :D